assets/src/components/activities/common/choices/ChoiceAuthoring.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { useState } from 'react';
+import { RichTextEditorConnected } from 'components/content/RichTextEditor';
+import {
+  Draggable,
+  DraggableStateSnapshot,
+  DraggingStyle,
+  NotDraggingStyle,
+} from 'react-beautiful-dnd';
+import { Choice, ChoiceId, RichText } from 'components/activities/types';
+import { RemoveButtonConnected } from 'components/activities/common/authoring/RemoveButton';
+
+interface Props {
+  canRemove: boolean;
+  icon: React.ReactNode;
+  index: number;
+
+  choice: Choice;
+  onEdit: (id: ChoiceId, content: RichText) => void;
+  onRemove: (id: ChoiceId) => void;
+}
+export const ChoiceAuthoringConnected: React.FC<Props> = ({
+  canRemove,
+  icon,
+  index,
+  choice,
+  onEdit,
+  onRemove,
+}) => {
+  const getStyle = (
+    style: DraggingStyle | NotDraggingStyle | undefined,
+    snapshot: DraggableStateSnapshot,
+  ) => {
+    const snapshotStyle = snapshot.draggingOver ? { 'pointer-events': 'none' } : {};
+    if (style?.transform) {
+      const axisLockY = `translate(0px, ${style.transform.split(',').pop()}`;
+      return {
+        ...style,
+        ...snapshotStyle,
+        minHeight: 41,
+        transform: axisLockY,
+      };
+    }
+    return {
+      ...style,
+      ...snapshotStyle,
+      minHeight: 41,
+    };
+  };
+
+  const [showDragIndicator, setShowDragIndicator] = useState(false);
+
+  return (
```

Comment:
This uses a drag and drop library, which is a different implementation from the current drag and drop setup we use in other places, but in all honesty is a lot easier to use and has some built-in settings I use here that would have taken awhile to build out.